interview-questions.md

```diff
@@ -7,59 +7,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a function? Why would you use one?
 
-  Your answer:
+  Your answer: A function is a line of code that can group several actions together.
 
-  Researched answer:
+  Researched answer: functions are a piece of code that has information used to return a specific ouput.
 
 
 
 2. What is the difference between console.log() and return?
 
-  Your answer:
+  Your answer: console.log executes the code and return gives you the output of the function.
 
-  Researched answer:
+  Researched answer: console.log gets the output of the code. return is the output of the function.
 
 
 
 3. In regards to functions, what is the difference between an argument and a parameter?
 
-  Your answer:
+  Your answer: argument is the input of your code and parameter is the output.
 
-  Researched answer:
+  Researched answer: Parameter is the placeholder and the argument is the content of your function.
 
 
 
 4. What is a built-in method?
 
-  Your answer:
+  Your answer: logging data types without the function.
 
   Researched answer:
 
 
 
 5. Give a brief description of proper pair programming techniques. What are the roles of each person?
 
-  Your answer:
+  Your answer: One person is driving, meaning they are doing the actual coding. The second person is navigating, meaning they guide the driver.
 
-  Researched answer:
+  Researched answer: Pretty much what I said in my own answer.
 
 
 
 6. What is something we did in class this week you found helpful?  
 
-  Your answer:
+  Your answer: Introduced graduates from the course that are currently working in the field. It gave me confidence to be able to continue even though this is very difficult.
 
```

Comment:
I'm glad to hear that is helpful. There will be more opportunities to meet LEARN alumni!